lib/service/service.go

```diff
@@ -828,7 +828,9 @@ func NewTeleport(cfg *Config) (*TeleportProcess, error) {
 		warnOnErr(process.closeImportedDescriptors(teleport.ComponentWindowsDesktop), process.log)
 	}
 
-	process.RegisterFunc("common.rotate", process.periodicSyncRotationState)
+	process.RegisterFunc("common.rotate", func() error {
+		return process.periodicSyncRotationState(cfg.RotationConnectionInterval, cfg.RestartThreshold)
```

Comment:
Doesn't `process` has a `cfg` field? Why did you elect to use parameters instead?

Reply:
Because I honestly didn't see it when I l;ooked. I can't have been looking that hard. Reverted.